gsonvalue/src/test/java/me/tatarka/gsonvalue/GsonValueDeserializeTest.java

```diff
@@ -39,6 +40,13 @@ public void setup() {
                 .create();
     }
 
+    @Test
+    public void deserializeWithNull() {
+        List<ConstructorArg> list = gson.fromJson("[null]", new TypeToken<List<ConstructorArg>>() {}.getType());
```

Comment:
Think it makes more sense to create a dedicated class with a null field for this test then using a collection.

Reply:
I can create one, but if I need the class for the write case(s) too, would you prefer that I duplicate the class(es) or place it the `me.tatarka.gsonvalue.model.deserialize` package and use that class for both the read and write cases?

Reply:
Put a class in both.